mps-sync-plugin-lib/src/main/kotlin/org/modelix/mps/sync/mps/notifications/INotifier.kt

```diff
@@ -0,0 +1,19 @@
+package org.modelix.mps.sync.mps.notifications
+
+import org.modelix.kotlin.utils.UnstableModelixFeature
+
+@UnstableModelixFeature(reason = "The new modelix MPS plugin is under construction", intendedFinalization = "2024.1")
+interface INotifier {
+
+    fun error(message: String, responseListener: UserResponseListener? = null)
+
+    fun warning(message: String, responseListener: UserResponseListener? = null)
+
+    fun info(message: String, responseListener: UserResponseListener? = null)
+}
+
+@UnstableModelixFeature(reason = "The new modelix MPS plugin is under construction", intendedFinalization = "2024.1")
+fun interface UserResponseListener {
+
+    fun userResponded(result: String)
```

Comment:
nit: consider enums or a sealed class hierarchy to communicate more clearly the possible values.

After looking at all the code, there is basically (1) a case for conformation (`result = "true"`)  and (2) everything else where the result is not relevant.

Possible enums code be: `USER_ACCEPTED`, `UNSPECIFIED`.

`USER_ACCEPTED` is also only possible from `AlertNotifier`.

Reply:
fixed in dd621a8a813107534fecbe17707fb87b2866dc08